runtime/ops/process.rs

```diff
@@ -285,7 +286,9 @@ fn spawn_child(
   // We want to kill child when it's closed
   command.kill_on_drop(true);
 
-  let mut child = command.spawn()?;
+  let mut child = command.spawn().with_context(|| {
+    format!("Failed to spawn: {:?}", command.as_std().get_program(),)
```

Comment:
```suggestion
    format!("Failed to spawn: {}", command.as_std().get_program().to_string_lossy())
```

Let's not use debug format for non-debugging related purposes because it's more error prone. If desiring quotes we can be explicit, but in this case it's probably better to not have quotes to allow copying and pasting more easily.